plugin/trino-redshift/src/test/java/io/trino/plugin/redshift/TestRedshiftCastPushdown.java

```diff
@@ -144,16 +154,25 @@ public void setupTable()
                 .addColumn("c_numeric_10_2", "numeric(10, 2)", asList(1.23, 22.67, null))
                 .addColumn("c_numeric_19_2", "numeric(19, 2)", asList(1.23, 22.67, null)) // Equal to REDSHIFT_DECIMAL_CUTOFF_PRECISION
                 .addColumn("c_numeric_30_2", "numeric(30, 2)", asList(1.23, 22.67, null))
+                .addColumn("c_char", "char", asList("'I'", "'P'", null))
```

Comment:
nit: for mismatch column case

```suggestion
                .addColumn("c_char", "char", asList("'I'", "'F'", null))
```

Reply:
Gentle reminder